vNext/test/NonSiloTests/project.json

```diff
@@ -1,5 +1,6 @@
 ﻿{
   "dependencies": {
+    "System.Management.Automation": "6.1.7601.17515",
```

Comment:
This package doesn't have any dependency. So I think it would fine running in ,NET core? https://www.nuget.org/packages/System.Management.Automation